api/calendar.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Calendar API
```

Comment:
It seems the API gives information about Holidays so rename `Calender` to `Holidays`
```suggestion
name: Holidays API
```